Userland/Libraries/LibJS/Bytecode/Generator.cpp

```diff
@@ -298,7 +298,7 @@ CodeGenerationErrorOr<void> Generator::emit_delete_reference(JS::ASTNode const&
 {
     if (is<Identifier>(node)) {
         auto& identifier = static_cast<Identifier const&>(node);
-        emit<Bytecode::Op::DeleteVariable>(intern_identifier(identifier.string()));
+        emit<Bytecode::Op::DeleteVariable>(intern_identifier(identifier.string()), identifier.is_local());
```

Comment:
Is this meaningfully different from using `Bytecode::Op::LoadImmediate` to load `Value(false)`?

Reply:
there would be no difference indeed!